docs/readmes/howtos/ue_ip_address_management.md

```diff
@@ -39,13 +39,27 @@ This mode will turn on the `uplink_br0` interface in the AGW to bridge UE
 traffic to the SGi interface. You will need to configure an internet gateway
 node that handles NAT and routing for all user plane traffic. The AGW will
 detect the (default) internet gateway from its networking stack and start ARP
-probes to get the gateway MAC address. This gateway MAC address is used as the
-target host for all bridged user plane traffic. If the AGW is not able complete
-ARP resolution, it floods user plane packets in SGi L2 segment. To avoid
-flooding SGi network, you can statically set the fallback MAC address of the
-internet gateway in `/var/opt/magma/configs/pipelined.yml` with the key
+probes to get the internet gateway MAC address. This gateway MAC address is
+used as the target host for all Non-NAT user plane traffic. If the AGW is not
+able complete ARP resolution, it floods user plane packets in SGi L2 segment.
+To avoid flooding SGi network, you can statically set the fallback MAC address
+of the internet gateway in `/var/opt/magma/configs/pipelined.yml` with the key
 `uplink_gw_mac`.
 
+### Router mode
+By default, AGW is in router mode. AGW acts as router for UE subnet. Note, there
+is no router advertisement support on AGW, so you need to manually configure the
+SGi IP as next hop for UE IP address subnet(s).
```

Comment:
Minor revision:
`
By default, the AGW is pre-configured in `router mode` where the AGW acts as the L3 next hop for the UE allocated subnet (UEs can be allocated IP addresses via the IP address pool or Static assignment).  Note that currently the AGW does not advertise this prefix dynamically so the upstream networks need to be configured with a static route pointing to the SGi interface for the UE subnet(s).
`